src/main/java/org/minions/devfund/danielmontecinos/CountCharacters.java

```diff
@@ -0,0 +1,26 @@
+package org.minions.devfund.danielmontecinos;
+
+/**
+ * My solution for kata https://www.codewars.com/kata/count-the-characters/.
+ */
+public final class CountCharacters {
+
+    /**
+     * Constructor.
+     */
+    public CountCharacters() {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1186)